docker-compose.yaml

```diff
@@ -53,6 +53,16 @@ services:
       orchestrator:
         condition: service_healthy
 
+  state-relay-server:
+    image: ghcr.io/espressosystems/espresso-sequencer/state-relay-server:main
+    ports:
+      - "$ESPRESSO_STATE_RELAY_SERVER_PORT:$ESPRESSO_STATE_RELAY_SERVER_PORT"
+    environment:
+      - $ESPRESSO_STATE_RELAY_SERVER_PORT
+      - $ESPRESSO_STATE_SIGNATURE_WEIGHT_THRESHOLD
```

Comment:
I may just not be familiar with the syntax but I don't think this should have a `$` in front?

Reply:
Right, removed in [1dde57c](https://github.com/EspressoSystems/espresso-sequencer/pull/1016/commits/1dde57c61e4683b1fde67bb5bde9e666a563311b)